docs/core/about.md

```diff
@@ -5,47 +5,45 @@ ms.date: 09/17/2019
 ---
 # About .NET Core
 
-.NET Core has the following characteristics:
+.NET Core is/has:
 
 - **Cross-platform:** Runs on Windows, macOS, and Linux [operating systems](https://github.com/dotnet/core/blob/master/os-lifecycle-policy.md).
-- **Consistent across architectures:** Runs your code with the same behavior on multiple architectures, including x64, x86, and ARM.
-- **Command-line tools:**  Includes easy-to-use command-line tools that can be used for local development and in continuous-integration scenarios.
-- **Flexible deployment:** Can be included in your app or installed side-by-side (user-wide or system-wide installations). Can be used with [Docker containers](docker/introduction.md).
-- **Compatible:** .NET Core is compatible with .NET Framework, Xamarin, and Mono, via [.NET Standard](../standard/net-standard.md).
-- **Open source:** The .NET Core platform is open source, using MIT and Apache 2 licenses. .NET Core is a [.NET Foundation](https://dotnetfoundation.org/) project.
-- **Supported by Microsoft:** .NET Core is supported by Microsoft, per [.NET Core Support](https://dotnet.microsoft.com/platform/support/policy).
+- **Open source:** The .NET Core platform is [open source](https://github.com/dotnet/core), using MIT and Apache 2 licenses. .NET Core is a [.NET Foundation](https://dotnetfoundation.org/) project.
+- **Modern:** It implements modern paradigms like asyncronous programming, no-copy patterns using structs, and resource governance for containers.
+- **High performance:** Uses [hardware intrinsics](https://devblogs.microsoft.com/dotnet/hardware-intrinsics-in-net-core/) and [constant investment](https://devblogs.microsoft.com/dotnet/performance-improvements-in-net-core-3-0/) to improve performance with every release.
```

Comment:
```suggestion
- **High performance:**  [Improves performance] (https://devblogs.microsoft.com/dotnet/performance-improvements-in-net-core-3-0/) with new features like [hardware intrinsics](https://devblogs.microsoft.com/dotnet/hardware-intrinsics-in-net-core/), [tiered compilation](https://github.com/dotnet/coreclr/blob/master/Documentation/design-docs/tiered-compilation.md), and [Span<T>](https://docs.microsoft.com/archive/msdn-magazine/2018/january/csharp-all-about-span-exploring-a-new-net-mainstay).
```

Reply:
Again, listing is not very homogeneous. While hardware intrinsics and `Span<T>` are things a developer can use, tiered compilation is the internal specifics of the runtime. Also, "new features" become old fast. Is `Span<T>` still new? Or the comparison ("improves", "new") is implicitly done with .NET Framework?

Reply:
Can remove "new". Agree.

All 3 of those things are performance-oriented investments we made that are intended to make your app faster. Hardware intrinsics and span are equally runtime features and they are also used extensively in the BCL so that they benefit you by default w/o you using them. So, I see these all as "same enough". Fair?

Reply:
> So, I see these all as "same enough". Fair?

I agree and admit I might have been too nit-picky.

By the way, the docs have already page about `Span<T>` and related types:
https://docs.microsoft.com/en-us/dotnet/standard/memory-and-spans/

Should it be linked instead of the magazine article?

Reply:
Good suggestion. good now?

Reply:
Yes, with two nits:
- Please remove the white space after "[high performance]". I'm afraid it breaks the link formatting now.
- Use the relative link: `../standard/memory-and-spans/index.md`. That way, the links are checked at the build.

Reply:
OK. Did that (I think).